.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to PyPi
+
+on: 
+  release:
+    types: [created]
+
+jobs:
+  build-and-publish:
+    name: Build python package and publish to PyPi
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
```

Comment:
No need to install `twine` explicitly, it is installed with the `uses: pypa/gh-action-pypi-publish@release/v1` line.